libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -104,4 +109,34 @@ export class MapUtilsService {
       return []
     }
   }
+
+  getLayerExtent(layer: MapContextLayerModel): Observable<Extent> {
+    let extent = []
+    if (layer.type === 'geojson' || layer.type === 'wfs') {
```

Comment:
WFS layers do not have the data inline, they rely on a url like WMS; for the datahub we don't use WFS layers, only geosjon and WMS

Reply:
ok, got it